.github/workflows/python-publish-aws-lambda-sdk.yml

```diff
@@ -0,0 +1,104 @@
+name: "Python: Publish python/serverless-aws-lambda-sdk"
+
+on:
+  push:
+    tags:
+      - "python/serverless-aws-lambda-sdk@[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  publishNewVersion:
+    name: Publish new version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Python and Pip
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+          # ensure project dependencies are cached
+          # When using only `pyproject.toml` for dependencies, see:
+          #  https://github.com/actions/setup-python/issues/529#issuecomment-1367029699
+          cache: 'pip'
+          cache-dependency-path: |
+            **/pyproject.toml
+
+      - name: Install main project dependencies
+        run: |
+          cd python/packages
+          python3 -m venv .venv
+          source .venv/bin/activate
+          python3 -m pip install ./aws-lambda-sdk --target=build
+
+      - name: Publish New Version on PyPI
+        env:
+          TWINE_USERNAME: "__token__"
+          TWINE_PASSWORD: ${{ secrets.PYPI_AUTH_TOKEN }}
+        run: |
+          cd python/packages
+          source .venv/bin/activate
+          cd ./aws-lambda-sdk
+
+          python3 -m pip install --upgrade build twine wheel
+          python3 -m build --wheel --sdist .
+          twine upload dist/*.tar.gz dist/*.whl
+
+      - name: Create lambda layer package
+        run: |
+          cd python/packages
+          source .venv/bin/activate
+
+          mkdir -p dist/{python/lib/python3.9/site-packages,sls-sdk-python}
+
+          cp -R ./build/* dist/python/lib/python3.9/site-packages
+
+          cp ./build/serverless_aws_lambda_sdk/internal_extension/__init__.py dist/sls-sdk-python
+          cp ./build/serverless_aws_lambda_sdk/internal_extension/base.py dist/sls-sdk-python
+          cp ./build/serverless_aws_lambda_sdk/internal_extension/exec_wrapper.py dist/sls-sdk-python
+          cp ./build/typing_extensions.py dist/sls-sdk-python
+          cp -R ./build/strenum dist/sls-sdk-python
+          cd dist
+          zip -r ../extension.internal.zip python sls-sdk-python
```

Comment:
In the next PR, with which we will start writing traces we will also set up integration tests, which will be run by Node.js.

Those tests should have a straightforward mean to build a layer (as for each integration test run we build layer upfront - so in all cases we're sure we're running testes against current code state).

So ideally if there's some script preconfigured which can then be run in context of Node.js integration tests, and ideally if it's used here, instead of those lengthy instructions set.

Reply:
> @selcukcihan that's outstanding work. Thanks for looking in detail into all those cases and sorting out approached ssues.
> 
> Cases that still bother me are:
> 
> * `6. Handler function specified but does not exist` - Why with internal extension error information: `HandlerNotFound: Handler 'foo' missing on module 'lambda_function'` is repeated twice?
> * `7. Handler function specified but is not a function - Here `TypeError: 'int' object is not callable`feels very different from`lambda_function.not_a_function is not a function`. It's like they reporting to two very different handler exports. Why is that?

👍  I'm working on these two cases specifically. I've found a way to make sure they behave the same. The logs are repeated twice because it seems Lambda Runtime is somehow retrying the whole exec_wrapper script. I'll amend the PR with the necessary fix and also post the latest results for these two cases.

Reply:
> 👍 I'm working on these two cases specifically. I've found a way to make sure they behave the same. The logs are repeated twice because it seems Lambda Runtime is somehow retrying the whole exec_wrapper script.

Yes, AWS Lambda when approaches failure on initialization does a retry. Still I wonder why it's not shown without the extension